reference/6/Microsoft.PowerShell.Utility/ConvertFrom-Csv.md

```diff
@@ -12,130 +12,148 @@ title: ConvertFrom-Csv
 ## EXAMPLES
 
 ### Example 1: Convert processes on the local computer to CSV format
-```
-PS C:\> $p = get-process | convertto-csv
-PS C:\> $p | convertfrom-csv
-```
 
-These commands convert the processes on the local computer into CSV format and then restore them to object form.
+This example shows how to convert the processes on the local computer into CSV format and then
+restore them to object form.
 
-The first command uses the **Get-Process** cmdlet to get the processes on the local computer.
-A pipeline operator (|) sends them to the **ConvertTo-CSV** cmdlet, which converts the process object to CSV format.
-The CSV strings are saved in the $p variable.
+```powershell
+$P = Get-Process | ConvertTo-Csv
+$P | ConvertFrom-Csv
+```
 
-The second command uses a pipeline operator to send the CSV strings in the $p variable to the **ConvertFrom-CSV** cmdlet.
-The cmdlet converts the CSV strings into CSV versions of the original process objects.
+The `Get-Process` cmdlet sends the processes down the pipeline to `ConvertTo-Csv`. The
+`ConvertTo-Csv` cmdlet converts the process objects to a series of CSV strings. The
+`ConvertFrom-Csv` cmdlet converts the CSV strings into CSV versions of the original process
+objects. The CSV strings are saved in the `$P` variable.
 
 ### Example 2: Convert a data object to CSV format and then to CSV object format
-```
-PS C:\> $Date = Get-Date | ConvertTo-Csv -Delimiter ";"
-PS C:\> ConvertFrom-Csv -InputObject $Date -Delimiter ";"
-```
 
-These commands convert a data object to CSV format and then to CSV object format.
+This example shows how to convert a data object to CSV format and then to CSV object format.
 
-The first command uses the Get-Date cmdlet to get the current date and time.
-A pipeline object (|) sends the date to the ConvertTo-Csv cmdlets, which converts the date object to a series of CSV strings.
-The command uses the *Delimiter* parameter to specify a semicolon delimiter.
-The strings are saved in the $date variable.
+```powershell
+$Date = Get-Date | ConvertTo-Csv -Delimiter ';'
+ConvertFrom-Csv -InputObject $Date -Delimiter ';'
+```
 
-The second command uses the **ConvertFrom-CSV** cmdlet to convert the CSV strings in the $date variable back to object format.
-The command uses the *InputObject* parameter to specify the CSV strings and the Delimiter parameter to specify the semicolon delimiter.
+The first command uses `Get-Date` to send the current date and time down the pipeline to
+`ConvertTo-Csv`. The `ConvertTo-Csv` cmdlet converts the date object to a series of CSV strings.
+The **Delimiter** parameter is used to specify a semicolon delimiter. The strings are saved in the
+`$Date` variable.
 
 ### Example 3: Use the header parameter to change the names of properties
+
+This example shows how to use the **Header** parameter of `ConvertFrom-Csv` to change the names of
+properties in the resulting imported object.
+
+```powershell
+$J = Start-Job -ScriptBlock { Get-Process } | ConvertTo-Csv
```

Comment:
Need to add the **-NoTypeInformation** parameter to `ConvertTo-Csv` for this example to work in all versions.

It is not needed for V6 but it is allowed. It is required pre-v6 for this example to work.

Reply:
Added -NoTypeInformation to the command and included a sentence in the example's explanation.